en-US/Players_Guide/03_Creating_a_Character/Level_Advancement/Multiclassing.md

```diff
@@ -0,0 +1,139 @@
+For instance, three Wizard levels and two Fighter levels make you a 5th-level character.
+
+As you level up, you may stay in your original class with a few levels in another, or change course entirely.
+You might even start progressing in a third or fourth class.
+Compared to a single-class character of the same level, you sacrifice focus for versatility.
+
+##### Prerequisites
+
+To qualify for a new class, meet the ability score prerequisites for both your current and new classes, as shown in the Multiclassing Prerequisites table.
+For instance, a Dreadnought multiclassing into Wodewose must have 13 or higher Strength and Wisdom scores.
+Without the training of a beginning character, you must be a quick learner with natural aptitude reflected by higher-than-average ability scores.
+
+_Multiclassing Prerequisites (table)_
+
+| Class       | Ability Score Minimum       |
+|:------------|:----------------------------|
+| Adept       | Dexterity 13 and Wisdom 13  |
+| Bard        | Charisma 13                 |
+| Cleric      | Wisdom 13                   |
+| Dreadnought | Strength 13                 |
+| Fighter     | Strength 13 or Dexterity 13 |
+| Ranger      | Dexterity 13 and Wisdom 13  |
+| Rogue       | Dexterity 13                |
+| Sorcerer    | Charisma 13                 |
+| Vanguard    | Strength 13 and Charisma 13 |
+| Warlock     | Charisma 13                 |
+| Wizard      | Intelligence 13             |
+| Wodewose    | Wisdom 13                   |
+
+##### Experience Points
+
+The experience point cost to gain a level is always based on your total character level, as shown in the Character Advancement table, not your level in a particular class.
+So, if you are a Cleric 6/Fighter 1, you must gain enough XP to reach 8th level before you can take your second level as a Fighter or your seventh level as a Cleric.
+
+##### Hit Points and Hit Dice
+
+You gain the hit points from your new class as described for levels after 1st.
+You gain the 1st-level hit points for a class only when you are a 1st-level character.
+
+You add together the Hit Dice granted by all your classes to form your pool of Hit Dice.
+If the Hit Dice are the same die type, you can simply pool them together.
+For example, both the Fighter and the Vanguard have a d10, so if you are a Vanguard 5/Fighter 5, you have ten d10 Hit Dice.
+If your classes give you Hit Dice of different types, keep track of them separately.
+If you are a Vanguard 5/Cleric 5, for example, you have five d10 Hit Dice and five d8 Hit Dice.
+
+##### Proficiency Bonus
+
+Your proficiency bonus is always based on your total character level, as shown in the [Character Advancement table in chapter XX](#Level_Advancement_level-advancement), not your level in a particular class.
+For example, if you are a Fighter 3/Rogue 2, you have the proficiency bonus of a 5th-level character, which is +3.
+
+<!-- TODO Add the multiclassing proficiencies table -->
```

Comment:
There are some open comments here; I can either wait for those to be resolved in the Google Doc, or transfer them to GitHub as well. Do you have a preference?